docs/source/index.rst

```diff
@@ -3,7 +3,16 @@ Documentation for ConStrain
 
 **Version**: |release|
 
+**ConStrain** is a data-driven knowledge-integrated framework that automatically verifies that controls function as intended. **ConStrain** is designed around three key features: building control knowledge integration, analytics, and automation. The framework includes three major components: a control verification algorithm library (rule-based, procedure-based, and AI-based), an automated preparation process and verification case generation, a standardized performance evaluation and reporting process.
+
+Applications of **ConStrain** include verifications of the implementation of control-related building energy code requirements (e.g., ASHRAE Standard 90.1) and sequences of operation (e.g., ASHRAE Guideline 36) using either data generated from building energy simulations and/or actual building monitoring system trend data.
+
 .. toctree::
    :maxdepth: 1
 
-   Code Documentation
\ No newline at end of file
+   Statement of Need
+   Quickstart Guide
+   Code Documentation
```

Comment:
We probably also need to add pages that document all the verification that are currently supported.

Reply:
Done!